.github/workflows/cmake.yml

```diff
@@ -0,0 +1,53 @@
+name: CMake
```

Comment:
```suggestion
name: CI
```

Reply:
This is the default name of the provided template :) I will rename it and also the file.

Reply:
done